src/routes/index.ts

```diff
@@ -161,7 +161,10 @@ router
     navigation.hasTrust,
   )
   .get(trustInvolved.get)
-  .post(trustInvolved.post);
+  .post(
+    ...validator.trustInvolved,
```

Comment:
For my own curiosity, what is the spread operator doing here?

Reply:
it is return set of validation middlewares
```
validator.trustInvolved = [
   validator1,
   validator2,
]

//. for this
 .post(
    ...validator.trustInvolved,
    post,
)
    
// analog
 .post(
   validator1,
   validator2,
   post,
)
```

Reply:
Awesome, thank you